src/Microsoft.Health.Fhir.Core/Features/Conformance/ConformanceProviderBase.cs

```diff
@@ -21,6 +21,11 @@ public abstract class ConformanceProviderBase : IConformanceProvider
 
         public abstract Task<ResourceElement> GetCapabilityStatementAsync(CancellationToken cancellationToken = default(CancellationToken));
 
+        internal void ClearCache()
+        {
+            _evaluatedQueries.Clear();
```

Comment:
@feordin I forgot about this, we should also clear this when we reflect changes in the capability statement to do with search params

Reply:
Separate bug?